website/docs/r/logic_app_integration_account_batch_configuration.html.markdown

```diff
@@ -0,0 +1,125 @@
+  resource_group_name      = azurerm_resource_group.example.name
+  integration_account_name = azurerm_logic_app_integration_account.example.name
+  batch_group_name         = "TestBatchGroup"
+
+  release_criteria {
+    message_count = 80
+  }
+}
+```
+
+## Arguments Reference
+
+The following arguments are supported:
+
+* `name` - (Required) The name which should be used for this Logic App Integration Account Batch Configuration. Changing this forces a new resource to be created.
+
+* `resource_group_name` - (Required) The name of the Resource Group where the Logic App Integration Account Batch Configuration should exist. Changing this forces a new resource to be created.
+
+* `integration_account_name` - (Required) The name of the Logic App Integration Account. Changing this forces a new resource to be created.
+
+* `batch_group_name` - (Required) The batch group name of the Logic App Integration Batch Configuration. Changing this forces a new resource to be created.
+
+* `release_criteria` - (Required) A `release_criteria` block as documented below.
+
+* `metadata` - (Optional) A JSON mapping of any Metadata for this Logic App Integration Account Batch Configuration.
+
+---
+
+A `release_criteria` block exports the following:
+
+* `batch_size` - (Optional) The batch size in bytes for the Logic App Integration Batch Configuration.
+
+* `message_count` - (Optional) The message count for the Logic App Integration Batch Configuration.
+
+* `recurrence` - (Optional) A `recurrence` block as documented below.
+
+---
+
+A `recurrence` block exports the following:
+
+* `frequency` - (Required) The frequency of the schedule. Possible values are `Day`, `Hour`, `Minute`, `Month`, `Second`, `Week` and `Year`.
+
+* `interval` - (Required) The number of `frequency`s between runs.
+
+* `end_time` - (Optional) The end time of the schedule.
+
+* `schedule` - (Optional) A `schedule` block as documented below.
+
+* `start_time` - (Optional) The start time of the schedule.
+
+* `time_zone` - (Optional) The timezone of the start/end time.
+
+---
+
+A `schedule` block exports the following:
+
+* `hours` - (Optional) A list containing a single item, which specifies the Hour interval at which this recurrence should be triggered.
+
+* `minutes` - (Optional) A list containing a single item which specifies the Minute interval at which this recurrence should be triggered.
+
+* `month_days` - (Optional) A list of days of the month that the job should execute on.
+
+* `monthly_occurrence` - (Optional) A `monthly_occurrence` block as documented below.
+
+* `week_days` - (Optional) A list of days of the week that the job should execute on. Possible values are `Sunday`, `Monday`, `Tuesday`, `Wednesday`, `Thursday`, `Friday` and `Saturday`.
+
+---
+
+A `monthly_occurrence` block exports the following:
+
+* `day` - (Required) The day of the occurrence. Possible values are `Sunday`, `Monday`, `Tuesday`, `Wednesday`, `Thursday`, `Friday` and `Saturday`.
```

Comment:
i think this might be more clear?
```suggestion
* `weekday` - (Required) The day of the occurrence. Possible values are `Sunday`, `Monday`, `Tuesday`, `Wednesday`, `Thursday`, `Friday` and `Saturday`.
```

Reply:
Updated